Source/radiation/RadSolve.cpp

```diff
@@ -1160,8 +1169,17 @@ void RadSolve::getEdgeMetric(int idim, const Geometry& geom, const Box& edgebox,
         geom.GetEdgeLoc(s, reg, I);
       }
       const Real *dx = geom.CellSize();
-      sphe(r.dataPtr(), s.dataPtr(), idim,
-           ARLIM(edgebox.loVect()), ARLIM(edgebox.hiVect()), dx);
+
```

Comment:
aren't we missing the 1-d case here, r = r**2?

Reply:
and the n = 0 case for multid?

Reply:
fixed, I think